src/coreclr/vm/stubmgr.cpp

```diff
@@ -1404,9 +1389,7 @@ static BOOL TraceManagedThunk(
 
     // Could be in the singlecast invoke stub (in which case the next destination is in _methodPtr) or a
```

Comment:
These comments need updating too

Reply:
I didn't update the comment because I'm not understanding the method.